### PR TITLE
Terminate action

### PR DESCRIPTION
**Pull request by lanwen**

### CODE REVIEW

**Review by eddumelendez — APPROVED** [review]
LGTM

**Review by ilopezluna — CHANGES_REQUESTED** [review]
I guess "test with defaults" should include `version=next`
Otherwise we can not test versions released but not published. In this case for example it try to define `--terminate` in the latest (not next) version. So the flag doesn't exists.

**Comment by lanwen** [comment]
@ilopezluna action should point to the current version in a stable state, but I'll also add the next version case. The current state shouldn't break, otherwise, we can break CI for users of the action, so the test was legit. I'll release the client once signed and restart tests of the action

**Review by ilopezluna — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -39,4 +45,5 @@ jobs:
 - `version` (_optional_):  `latest` (default) - Testcontainers Cloud binary version
 - `wait` (_optional_):  `true` (default) /`false` - if action should wait until agent successfully connects to Testcontainers Cloud in advance
 - `args` (_optional_): `string` - flags/arguments of the agent to pass as is. Consult with the knowledge base to find out more about possible options.
-- `logfile` (_optional_): `string` - file to write the agent output instead of the standard out.
\ No newline at end of file
+- `logfile` (_optional_): `string` - file to write the agent output instead of the standard out.
+- `action` (_optional_): `prepare` (default) / `terminate` - action could be specified explicitly to terminate sessions eagerly
```
Comment (lanwen) [inline README.md:49]:
```suggestion
- `action` (_optional_): `prepare` (default) / `terminate` - action could be specified explicitly to terminate sessions eagerly.
```

**Pull request merged**